CHANGELOG.md

```diff
@@ -19,10 +19,12 @@
   @weiglszonja (#1403)
 - Improved constructor docstrings for Image types. @weiglszonja (#1418)
 - Added documentation for exporting NWB files. @rly (#1417)
+- Improved documentation formatting. @bendichter (#1438)
+- Minor text fixes. @bendichter (#1437, #1400)
 
```

Comment:
I think the Changelog may be missing the updates to the tutorial that were merged today

Reply:
The tutorial updates are included in the list items above this. The two items added here were missing.